frontend/src/components/datasets/DatasetsPage.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Redirect, Route, Switch } from "react-router-dom"
+import PageContainer from "../PageContainer"
+import DatasetDetailContent from "./DatasetDetailContent";
+import DatasetsListContent from "./DatasetsListContent";
+
+const DatasetsPage = () => <PageContainer>
+    <Switch>
```

Comment:
Might be worth it to merge your other PR and update the content here to be compatible with the new react requirements.